packages/components/AssetsPagination.tsx

```diff
@@ -0,0 +1,295 @@
+      </View>
+      <View
+        style={{
+          flexDirection: "row",
+          alignItems: "center",
+          justifyContent: "flex-start",
+        }}
+      >
+        <BrandText
+          style={{
+            ...fontSemibold14,
+            color: neutral77,
+            paddingRight: layout.spacing_x1,
+            lineHeight: 14,
+          }}
+        >
+          | Go to page:
+        </BrandText>
+        <TertiaryBox
+          style={[
+            {
+              flexDirection: "row",
+              paddingHorizontal: layout.spacing_x1_5,
+              backgroundColor: neutral17,
+              width: 80,
+              height: SEARCH_BAR_INPUT_HEIGHT,
+              alignItems: "center",
+            },
+          ]}
+        >
+          <TextInput
+            defaultValue={(currentPage + 1).toString()}
+            inputMode="numeric"
+            style={[
+              fontSemibold14,
+              {
+                color: "white",
+                flex: 1,
+                width: 56,
+              },
+              { outlineStyle: "none" } as any,
+            ]}
+            onSubmitEditing={(value) => {
+              onChangePage(+value.nativeEvent.text - 1);
+            }}
+          />
+        </TertiaryBox>
+      </View>
+    </View>
+  );
+};
+
+const RightContainer = ({
+  currentPage,
+  maxPage,
+  onChangePage,
+}: PaginationProps) => {
+  return (
+    <View style={{ flex: 1 }}>
+      <View
+        style={{
+          flexDirection: "row",
+          alignItems: "center",
+          justifyContent: "center",
+        }}
+      >
+        <TouchableOpacity onPress={() => onChangePage(0)}>
+          <TertiaryBox
+            style={{
+              height: 42,
+              width: 56,
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <SVG source={chevronLeftDoubleSVG} height={16} width={16} />
+          </TertiaryBox>
+        </TouchableOpacity>
+        <SpacerRow size={1} />
+
+        <TouchableOpacity onPress={() => onChangePage(currentPage - 1)}>
+          <TertiaryBox
+            style={{
+              height: 42,
+              width: 56,
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <SVG source={chevronLeftSVG} height={16} width={16} />
+          </TertiaryBox>
+        </TouchableOpacity>
+        <SpacerRow size={2} />
+
+        <View
+          style={{
+            flexDirection: "row",
+            alignItems: "center",
+            justifyContent: "flex-end",
+          }}
+        >
+          <TouchableOpacity
+            style={{
+              flexDirection: "row",
+              alignItems: "center",
+            }}
+            onPress={() => {
+              onChangePage(
+                currentPage + 1 >= maxPage ? currentPage - 1 : currentPage,
+              );
+            }}
+          >
+            <Box
```

Comment:
Please don't use directly Box components it's more a base component for 3 other consistant components that are PrimaryBox, SecondaryBox and TertiaryBox so please use one them 👍

Reply:
Okay, I will fix.